Update bulk data examples (#209)
openfisca/openfisca-doc: source/simulate/index.md

Nevertheless, you won't have the same experience defining those various situations sizes and linking them to your simulation. So, multiple options could be used to describe this information:

- either [test cases](./run-simulation.md#test-cases): you simulate the legislation for a small number of persons,
- or [bulk data](./run-simulation.md#data): you provide a population (survey with aggregated data, CSV files with bulk data, etc.) on which you want to apply the legislation.

> Then, when you have a defined situation and need to replicate an input in a range of values (e.g. a `salary` in [0, 100 000]), you can define an axe as described in [the axes section](./replicate-simulation-inputs.md).

Reviewer: an axe -> an axis ("axe" c'est une hache, en anglais "axis" a un pluriel irrégulier)

> Finally, you might also want to understand how your simulation result was calculated. To trace its steps, you can use the `trace` functions as described in [this section](./analyse-simulation.md).

Reviewer 2: ```suggestion
Finally, you might want to understand how your simulation result was calculated. To do so, you can use the `trace` feature described in [this section](./analyse-simulation.md).
```
